ResumptionScenarios.md

```diff
@@ -0,0 +1,108 @@
+# Resumption Scenarios
+
+This [protocol](./protocol.md) addition, shows possible resumption scenarios to ensure that the protocol is understood properly and an implementer may see all posible scenarios which
+
+```
+Client                                                                                           | Server 
+1                                                                                                |
```

Comment:
Can we add numbers on the Server side, either replacing or in addition to the vertical bars, just like the line numbers on the Client side. It will make it easier to match.